crates/meroctl/src/cli/context/identity.rs

```diff
@@ -182,3 +198,28 @@ async fn list_identities(
     )
     .await
 }
+async fn identity_exists_in_context(
+    multiaddr: &Multiaddr,
+    client: &Client,
+    keypair: &Keypair,
+    context: &Alias<ContextId>,
+    target_identity: &PublicKey,
+) -> EyreResult<bool> {
+    let context_id = resolve_alias(multiaddr, keypair, context.clone(), None)
```

Comment:
`Alias<T>: Copy`

https://github.com/calimero-network/core/blob/d513acff5d67850ddf581b511bb6f3074153ec75/crates/primitives/src/alias.rs#L44

```suggestion
    let context_id = resolve_alias(multiaddr, keypair, *context, None)
```